.github/workflows/copr_build.yml

```diff
@@ -0,0 +1,92 @@
+name: copr
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+env:
+  COPR_ACCOUNT: '@sssd'
+  COPR_PROJECT: pr${{ github.event.pull_request.number }}
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      srpm: ${{ steps.srpm.outputs.file }}
+      chroots_json: ${{ steps.chroots.outputs.json }}
+    permissions:
+      contents: read
+    steps:
+    - name: Checkout source
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+        repository: ${{ github.event.pull_request.head.repo.full_name }}
+    - name: Build source rpm
+      id: srpm
+      uses: ./.github/actions/build-srpm
```

Comment:
IMO it is a bit confusing to call `.github/actions/build-srpm`, which acts as a wrapper for `https://github.com/SSSD/action-build-srpm`

It would be great if these steps could all be self-contained in `github.com/SSSD/action-build-srpm`, the fact that it is in SSSD github org implies it is for use only with SSSD. If that is not feasible then I propose to at least include `wrapper` somewhere in the name of  `.github/actions/build-srpm/action.yml`

Reply:
Okay that works.

Is it correct that actions created inside SSSD repo `.github/actions/` files are re-usable for internal use by the SSSD project only, and `github.com/SSSD/action*` for external/generic use?

Reply:
Yes. Even .github/actions can be reused outside sssd, but they are not meant to.